public/app/features/migrate-to-cloud/onprem/NameCell.tsx

```diff
@@ -35,6 +35,8 @@ function ResourceInfo({ data }: { data: ResourceTableItem }) {
       return <DatasourceInfo data={data} />;
     case 'FOLDER':
       return <FolderInfo data={data} />;
+    case 'LIBRARY_ELEMENT':
+      return null;
```

Comment:
I'm just curious - did you get a type error due to non-exhaustive switch, or just added this in anyway?

Reply:
> I'm just curious - did you get a type error due to non-exhaustive switch, or just added this in anyway?

I got a type error indeed, and VSCode gave me the hint as well :)
![Screenshot 2024-09-26 at 14 46 53](https://github.com/user-attachments/assets/6778f52a-e16d-4b4a-a9cd-35bd39e39eb0)